controllers/apply-next/simple/fields.js

```diff
@@ -405,20 +438,44 @@ const allFields = {
         name: 'charity-number',
         label: { en: 'Charity registration number', cy: '' },
         explanation: {
-            en:
-                'If you are registered with OSCR, you only need to provide the last five digits of your registration number.',
+            en: `If you are registered with OSCR, you only need to provide the last five digits of your registration number.`,
             cy: ''
         },
         type: 'text',
-        attributes: {
-            size: 20
-        },
-        isRequired: true,
-        schema: Joi.any().when('organisation-type', {
-            is: Joi.valid(organisationTypes.unincorporatedRegisteredCharity.value),
-            then: Joi.number().required()
+        attributes: { size: 20 },
+        isRequired(formData = {}) {
```

Comment:
I've allowed `isRequired` to be a function so that we can conditionally determine whether a field should be marked as required on the front-end. Duplicating the field didn't work as the two schema's clashed with one another.